lib/collection/view/second_pass_help_page.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:flutter/material.dart';
+
+class SecondPassHelpPage extends StatelessWidget {
+  const SecondPassHelpPage({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        appBar: AppBar(title: const Text('Second Pass Help')),
+        body: Column(children: const [
+          Padding(
+            padding: EdgeInsets.all(8.0),
+            // TODO: refactor into something nicer
+            child: Text(
+                'Please choose the source of food, and form when eaten, '
+                'and fill in a detailed description of each food item consumed.\n\n'
+                'Use the probe list below to ask the respondent for more information.\n\n'),
+            // TODO: add probe list, maybe add search, or perhaps make the probe
+            // questions automatic in second pass screen based on the chosen food
```

Comment:
Not a coding implementation comment but might be worth rethinking the concept of help pages in the context of a digital form - having it in a separate page might be a bit disconnected for purposes of probing

Reply:
Yeah, I think this is quite a personal preference type of design decision. We should ask ICRISAT about it, I think for now it's alright to leave it like it is, and once we get feedback we can change it as requested 😊.